sabnzbd/assembler.py

```diff
@@ -207,7 +203,7 @@ def assemble(nzo: NzbObject, nzf: NzbFile, file_done: bool):
         # Final steps
         if file_done:
             set_permissions(nzf.filepath)
-            nzf.md5sum = nzf.md5.digest()
+            nzf.md5sum = nzf.crc32
```

Comment:
Can also probably be renamed to `crc32`? Or are there spots where you have doubts if it will be ok?